### PR TITLE
Add support for metric prefix

### PR DESCRIPTION
**Pull request by skygeo**
Prepend every metrics with a custom prefix set in the toml configuration file.

### CODE REVIEW

**Comment by skygeo** [comment]
Hi @kylebrandt @mjibson @gbrayut @Dieterbe
Is there a process for reviews? Or raising features like the ones proposed here?

Also, I am not sure why the tests are failing here, it seems unrelated. 

Let me know if I can help in any ways.
Many Thx

File: cmd/scollector/conf/conf.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ type Conf struct {
 	// UseSWbemServicesClient specifies if the wmi package should use SWbemServices.
 	UseSWbemServicesClient bool
 
+	// Prefix prepended to all metrics path
+	Prefix string
```
Comment (captncraig) [inline cmd/scollector/conf/conf.go]: Let's call this "MetricPrefix" to be more explicit.

File: cmd/scollector/collectors/collectors.go [hunk 2]
```diff
@@ -296,6 +301,9 @@ func AddTS(md *opentsdb.MultiDataPoint, name string, ts int64, value interface{}
 // automatically added. If the value of the host key is the empty string, it
 // will be removed (use this to prevent the normal auto-adding of the host tag).
 func Add(md *opentsdb.MultiDataPoint, name string, value interface{}, t opentsdb.TagSet, rate metadata.RateType, unit metadata.Unit, desc string) {
+	if Prefix != "" {
```
Comment (captncraig) [inline cmd/scollector/collectors/collectors.go]: This code needs to be in `AddTS` instead of `Add`, so it works for everything.

**Comment by captncraig** [comment]
It looks like the build is failing because `gofmt` needs to be run.

**Comment by skygeo** [comment]
Hi @captncraig ! Thank's for your input.
+ The argument is now "MetricPrefix" instead of "Prefix".
+ Moved the code in the "AddTS" method instead of "Add" method.
+ Added some documentation.
+ Travis Build now passing.

Let me know if you need anything else.

**Comment by skygeo** [comment]
Any news on that? @captncraig  ??

**Comment by captncraig** [comment]
Sorry, this looks good now. Only thing left is a mention in  the documentation (https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/doc.go)

**Comment by captncraig** [comment]
Looks great. Thanks.

**Pull request merged**